src/scripts/mangrove-watch-images/load.js

```diff
@@ -0,0 +1,150 @@
+const fs = require('fs');
+const fsp = require('fs/promises');
+
+const FormData = require('form-data');
+const csv = require('csvtojson');
+const axios = require('axios').default;
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+
+// Parameter parsing & check
+const parameters = ['dataResourceId'];
+const args = yargs(hideBin(process.argv)).argv;
+for (const param of parameters) {
+  if (!args[param]) {
+    console.log(`Missing '${param}' parameter!`);
+    return;
+  }
+}
+
+async function uploadImage(image, cookie, imagesDir = 'images') {
+  // Create a new FormData Instance
+  const form = new FormData();
+
+  // Append form attributes
```

Comment:
Rights field is missing. Add `Fair use with acknowledgement of rights holder.` to it.

Reply:
Added